### PR TITLE
Updating status bar values

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
Semantic name clean up and mapping for status bar 
#### Icons: 
 image

#### Backgrounds: 
image

#### Data viz colors
image

#### All Data and Status Semantic Names in Light Theme:
dataColor1: "#0078D4"
dataColor2: "#EF6950"
dataColor3: "#00188F"
dataColor4: "#00A2AD"
dataColor5: "#4B003F"
dataColor6: "#E3008C"
dataColor7: "#022F22"
dataColor8: "#917EDB"
dataColor9: "#001D3F"
dataColor10: "#502006"
noData2Color: "#747474"
noDataColor1: "rgba(128, 128, 128, .70)"
statusDefaultBackground: "#F0F6FF"
statusDefaultBorder: "#F0F6FF"
statusDisabledIcon: "#8A8886"
statusErrorBackground: "#FEF0F1"
statusErrorBorder: "#FEF0F1"
statusErrorIcon: "#a4262c"
statusErrorText: "#292827"
statusInformationBackground: "#F0F6FF"
statusInformationIcon: "#015cda"
statusInformationText: "#292827"
statusLink: "#005A9E"
statusLinkHover: "#004578"
statusSuccessBackground: "#F8FFF0"
statusSuccessBorder: "#F8FFF0"
statusSuccessIcon: "#57A300"
statusSuccessText: "#292827"
statusUpsellBackground: "#F9F3FC"
statusUpsellIcon: "#8a2da5"
statusWarningBackground: "#FFF8F0"
statusWarningBorder: "#FFF8F0"
statusWarningIcon: "#DB7500"
statusWarningText: "#292827"

#### All Data and Status Semantic Names in Dark Theme:
dataColor1: "#0078D4"
dataColor2: "#CB4936"
dataColor3: "#BEDFFF"
dataColor4: "#038387"
dataColor5: "#F6C0FF"
dataColor6: "#D90086"
dataColor7: "#A4E1D2"
dataColor8: "#6E5DB7"
dataColor9: "#C7D3FF"
dataColor10: "#FFCA8A"
noData2Color: "#747474"
noDataColor1: "rgba(128, 128, 128, .70)"
statusDefaultBackground: "#00245B"
statusDefaultBorder: "#00245B"
statusDisabledIcon: "#8A8886"
statusErrorBackground: "#442726"
statusErrorBorder: "#442726"
statusErrorIcon: "#e00b1c"
statusInformationBackground: "#00245B"
statusInformationIcon: "#015cda"
statusInformationText: "#F3F2F1"
statusLink: "#6cb8f6"
statusLinkHover: "#82C7FF"
statusSuccessBackground: "#393D1B"
statusSuccessBorder: "#393D1B"
statusSuccessIcon: "#57A300"
statusSuccessText: "#F3F2F1"
statusUpsellBackground: "#38134A"
statusUpsellIcon: "#8a2da5"
statusWarningBackground: "#4F2A0F"
statusWarningBorder: "#4F2A0F"
statusWarningIcon: "#DB7500"
statusWarningText: "#F3F2F1"

#### Notes
Light semantic names listed here are the same in high contrast light, and dark semantic names are the same in high contrast dark.
NoData1 and NoData2 is the same in all 4 subthemes

#### Maintained status bar styles
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/34af1fe15779b6069c9081d5ad60cbb0289746b3'>34af1fe15779b6069c9081d5ad60cbb0289746b3</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: change/@fluentui-azure-themes-ebf457c8-3a67-4f54-8eb2-86c27a79810f.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-azure-themes-ebf457c8-3a67-4f54-8eb2-86c27a79810f.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28440&commitId=1f1e6f4a89f40617a6df6fc8736d8c890e2ef2f2&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by Jacqueline-ms** [comment]
@robtaft-ms ready for review when you are :)

File: packages/azure-themes/src/azure/AzureColors.ts [hunk 2]
```diff
@@ -198,12 +197,12 @@ export const DarkSemanticColors: IAzureSemanticColors = {
       upsell: BaseColors.PURPLE_38134A,
     },
     icon: {
-      default: BaseColors.BLUE_2899F5,
-      okay: BaseColors.GREEN_92C353,
```
Comment (Jacqueline-ms) [inline packages/azure-themes/src/azure/AzureColors.ts:202]: Find a reference for these dark colors

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by robtaft-ms — APPROVED** [review]
This looks great, thank you so much!

**Pull request merged**